### DISCUSSION TITLE
Karavan Installation to Kubernetes

### DISCUSSION THREAD

**Discussion by ravishankar-hassain** [apache/camel-karavan#1049] [Q&A] [answered]
How to install Karavan in a Kubernetes cluster (not minikube) running either in the public cloud / on-prem using the Karavan CLI? Is there any documentation for the same?

**Comment by mgubaidullin** [comment]
It depends on your Kubernetes, Container Registry, Git repository, however procedure is the same as minikube.
Another way is to save YAML generated by Karavan-installer and customize it according to your infrastructure.

**Reply by bitsInnovate00** [reply]
Can you please give more insights on the second point. How to save YAML generated by Karavan-installer ?

**Comment by ravishankar-hassain** [comment]
> It depends on your Kubernetes, Container Registry, Git repository, however procedure is the same as minikube. Another way is to save YAML generated by Karavan-installer and customize it according to your infrastructure.

Can I use the Karavan CLI with options to do that by specifying my kubernetes cluster details. Also, on the second point how to save  YAML generated by Karavan-installer

**Comment by mgubaidullin** [comment, marked as answer]
https://github.com/apache/camel-karavan/blob/main/docs/WEB_KUBERNETES.md#install-parameters
